scripts/ci/qemu-run.sh

```diff
@@ -52,11 +52,15 @@ This is the QEMU wrapper script for the Dasharo Open Source Firmware Validation.
 
   Environmental variables:
     DIR         working directory, defaults to current working directory
+    DRIVE_PATH  path to file which will be added as a virtual hard drive. The file
+                should match formats described in chapter Disk Images of QEMU
+                documentation
 
 Example usage:
     ./$(basename $0) vnc firmware
     ./$(basename $0) graphic os_install
     DIR=/my/work/dir ./$(basename $0) graphic os
+    DRIVE_PATH=/path/to/image ./$(basename $0) graphic os
```

Comment:
How it should be different from HDD_PATH?

Reply:
This was added to boot qemu with two images, Ubuntu installed on HDD_PATH and DTS added via DRIVE_PATH. @PLangowski I think you worked on that. Maybe we could use here some different logic to be able to mount multiple images?